pkg/utils/topology/scheduling.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 PingCAP, Inc. Licensed under Apache-2.0.
+
+package topology
+
+import (
+	"context"
+	"encoding/json"
+	"sort"
+
+	"github.com/pingcap/tidb-dashboard/pkg/pd"
+	"github.com/pingcap/tidb-dashboard/util/distro"
+	"github.com/pingcap/tidb-dashboard/util/netutil"
+)
+
+func FetchSchedulingTopology(ctx context.Context, pdClient *pd.Client) ([]SchedulingInfo, error) {
+	nodes := make([]SchedulingInfo, 0)
+	data, err := pdClient.WithoutPrefix().SendGetRequest("pd/api/v2/ms/members/scheduling")
```

Comment:
miss the `/` at the beginning, else it reports the following errors:

![image](https://github.com/pingcap/tidb-dashboard/assets/1284531/6672bc83-dcc3-4df3-be74-ebafbbef2f03)